canvas/templatetags/canvas.py

```diff
@@ -40,3 +40,11 @@ def event_button_text(event, user):
         return "Not Available"
     else:
         return "Open"
+
+
+@register.simple_tag
+def tokens_column_name(event):
+    if event != '' and event.is_exam_and_open():
```

Comment:
I don't get the event != '' part. I think you have check if the event is an instance of Event.

Reply:
Good catch, I'll do that